packages/lab/stories/pill-next/pill-next.qa.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import { PillNext } from "@salt-ds/lab";
+import { ComponentMeta, Story } from "@storybook/react";
+import { QAContainer, QAContainerProps } from "docs/components";
+import { FavoriteIcon } from "@salt-ds/icons";
+
+export default {
+  title: "Lab/Pill Next/QA",
+  component: PillNext,
+} as ComponentMeta<typeof PillNext>;
+
+const noop = () => undefined;
+
+export const ExamplesGrid: Story<QAContainerProps> = (props) => {
+  const { className, ...rest } = props;
+  return (
+    <QAContainer cols={3} height={300} itemPadding={3} width={1300} {...rest}>
+      <PillNext className={className} onClick={noop}>
+        Default Pill
+      </PillNext>
+      <PillNext className={className} onClick={noop} disabled>
+        Disabled Pill
+      </PillNext>
+      <PillNext className={className} icon={<FavoriteIcon />} onClick={noop}>
+        With Icon Pill
+      </PillNext>
+      <PillNext onClick={noop} className={className}>
```

Comment:
don't think you need to pass className in these examples. Do you?

Reply:
Probably not 😁